README.md

```diff
@@ -1,2 +1,2 @@
 ## Overview
-The `parse-config.sh` script is used as an init container to create a `.env` file from other config files. In our current setup, these other config files are K8S secrets mounted as volumes. The format of the resulting `.env` file is a traditional `KEY=VALUE` text file, with each entry being seperated by a new line. This way, applications need to be only aware of one file to load its configuration. Aside from the bundling, the script also formats the key to standardise naming.
+The `parse-config.sh` script is used as an init container to create a `application.properties` file from other config files. In our current setup, these other config files are K8S secrets mounted as volumes. The format of the resulting `application.properties` file is a `key.subkey=value` text file, with each entry being seperated by a new line. This way, applications need to be only aware of one file to load its configuration. Aside from the bundling, the script also formats the key to standardise naming.
```

Comment:
Do you want more than this @bpesics? I'm thinking the rest of the docs (lifecycle & how to add properties, etc) should live in SER website.